app/serializers/api/v3/ecf/partnership_serializer.rb

```diff
@@ -51,7 +51,12 @@ class PartnershipSerializer
         end
 
         attribute :updated_at do |partnership|
-          partnership.updated_at.rfc3339
+          [
+            partnership.updated_at,
+            partnership.school.updated_at,
+            partnership.delivery_partner&.updated_at,
+            partnership.school&.induction_coordinator_profiles&.first&.updated_at,
```

Comment:
could we use `partnership.school&.induction_coordinators&.first&.updated_at` instead as that's what's being updated (the user)?

Reply:
fixed 👍